packages/maskbook/src/plugins/Wallet/services/keystore.ts

```diff
@@ -0,0 +1,86 @@
+import scrypt from 'scrypt-js'
+import * as crypto from 'crypto'
```

Comment:
Please DO NOT use the `crypto` comes from the NodeJS std. Use Web Crypto API instead